src/api/DataApi.js

```diff
@@ -688,6 +689,55 @@ function updateEntityData(
     });
 }
 
+/**
+ * `POST /data/binary`
+ *
+ * Loads a presigned URL for a particular binary object with the requested content disposition
+ *
+ * @static
+ * @memberof lattice.DATA_API
+ * @param {Object} binaryObjectRequest
+ * @returns {Promise<Object>} - a Promise that resolves the same request structure, with the
+ * content disposition field replaced by a presigned URL for the requested binary object, with the
+ * specified content disposition
+ *
+ * @example
+ * DataApi.loadBinaryProperties(
+ *  {
+ *    "value": {
+ *      "843fbf13-1857-4786-ac84-fd15b9f83b0b": { // entitySetId
+ *        "3d810000-0000-0000-8000-000000025d14": { // entityKeyId
+ *          "5364cb1b-ecf4-459d-b8d4-99b47b31281c": { // propertyTypeId
+ *            // s3 digest: content disposition
+ *            "500534a6a1072dfff31f4bc50e7154e8": "attachment; filename=\"filename.jpg\""
+ *          }
+ *        }
+ *      }
+ *    }
+ *  }
+ * );
+ */
+
+function loadBinaryProperties(
```

Comment:
`lattice-js` conventions are to use `get` instead of `load`/`fetch`. Please change `load` to `get`.